python/tests/test_nn.py

```diff
@@ -378,6 +378,49 @@ def test_log_sigmoid(self):
         self.assertEqual(y.shape, [3])
         self.assertEqual(y.dtype, mx.float32)
 
+    def test_step_activation(self):
+        x = mx.arange(-3, 4)
+        expected = mx.array([0, 0, 0, 0, 0, 1, 1])
+        y = nn.Step()(x)
+        self.assertTrue(mx.array_equal(y, expected))
+
+        y = nn.Step(2)(x)
+        expected = mx.array([0, 0, 0, 0, 0, 0, 1])
+        self.assertTrue(mx.array_equal(y, expected))
+
+    def test_selu(self):
+        x = mx.arange(-3, 4)
+        expected = mx.array(
+            [
+                -1.670563817024231,
+                -1.5201621055603027,
+                -1.1113275289535522,
+                0.0,
+                1.0506999492645264,
+                2.1013998985290527,
+                3.152099847793579,
+            ]
+        )
+        y = nn.SELU()(x)
+        self.assertTrue(mx.array_equal(y, expected))
+
+    def test_swish(self):
+        x = mx.arange(-3, 4)
+        expected = mx.array(
+            [
+                -0.14227759838104248,
+                -0.23840594291687012,
+                -0.26894140243530273,
+                0.0,
+                0.7310585975646973,
+                1.7615940570831299,
+                2.857722282409668,
+            ]
+        )
+        y = nn.Swish()(x)
+        mx.array_equal(y, expected)
+        self.assertTrue(mx.array_equal(y, expected))
```

Comment:
Same comment as above re `mx.allclose`. Also the first `array_equal` is not used, could you remove it?